.github/workflows/build.yml

```diff
@@ -43,3 +43,15 @@ jobs:
             ./lib/ryzenadj.h
             ./prebuilt/WinRing0x64.dll
             ./prebuilt/WinRing0x64.sys
+
+  Linux:
+    runs-on: Ubuntu-20.04
```

Comment:
does not use latest because Ubuntu-latest has a warning message to tell anybody about migration about the migration to 20.04 at the end of march